paws.common/R/net.R

```diff
@@ -90,18 +92,20 @@ issue <- function(http_request) {
   url <- build_url(http_request$url)
   headers <- unlist(http_request$header)
   body <- http_request$body
-  timeout <- if (!is.null(http_request$timeout)) httr::timeout(http_request$timeout)
+  timeout <- if (!is.null(http_request$timeout)) httr::config(connecttimeout = http_request$timeout)
+  times <- if (!is.null(http_request$max_attempts)) http_request$max_attempts else 1
```

Comment:
This inline if-else statement isn't idiomatic. Maybe we should instead use:
`times <- ifelse(!is.null(http_request$max_attempts), http_request$max_attempts, 1)`

Reply:
We also default it to 1 anyway

Reply:
Addressed in a slightly different way. Let me know what you think. I kept the null guards since we can't guarantee the function will get non-nulls.